scss/components/_contact-avatar.scss

```diff
@@ -0,0 +1,4 @@
+.civicase__contact-avatar {
+  background: #99c6e5;
```

Comment:
Can we use variable here? It would be great if we can pick a closer shade of this color from the variables already defined.

Reply:
Added variable. There is no closer shade here unfortunately.